src/imapcommands.c

```diff
@@ -2405,7 +2405,20 @@ static void _ic_store_enter(dm_thread_data *D)
 	if (self->ids_list) {
 		GString *failed_ids = g_list_join_u64(self->ids_list, ",");
 		buffer = p_string_new(self->pool, "");
-		p_string_printf(buffer, "MODIFIED [%s]", failed_ids->str);
+		Field_T val;
+		GETCONFIGVALUE("compatibility_report_skip_failed", "IMAP", val);
+		if (strlen(val)){
```

Comment:
could you please follow the convention used elsewhere in the code with regards to spaces before '{' and after '}' when followed be a keyword